wiki-docs/docs/interfaces/validators/validator-set-base.md

```diff
@@ -0,0 +1,68 @@
+The `IChildValidatorSetBase` interface is part of the childchain. It handles validator registration, stake storage, and reward distribution. This user guide will explain the key components and functions of the interface.
```

Comment:
This should be updated in accordance to these two contracts:

https://github.com/0xPolygon/core-contracts/blob/dev/contracts/child/validator/ValidatorSet.sol
https://github.com/0xPolygon/core-contracts/blob/dev/contracts/child/validator/RewardPool.sol

Reply:
can we create a ticket for this as we don't have time now to do new content please? https://polygon.atlassian.net/jira/your-work please tag me and elaborate a little